deps/node-inspect/lib/_inspect.js

```diff
@@ -39,7 +39,7 @@ const [ InspectClient, createRepl ] =
 
 const debuglog = util.debuglog('inspect');
 
-exports.port = 9229;
```

Comment:
I assume this is to support `node --{inspect,debug}-port=1234 {inspect,debug} my-script.js` etc.? If so, I'd rather make that explicit.